swaggerhub-wso2-integration/src/main/java/com/swagger/plugins/wso2/MakeHttpRequestService.java

```diff
@@ -0,0 +1,126 @@
+        postRequest.setHeader("Content-Type", contentType);
+        postRequest.setEntity(payload);
+        try {
+            response = httpClient.execute(postRequest);
+        } catch (IOException e) {
+            log.error("Error making POST request", e);
+            throw new com.smartbear.swaggerhub.plugins.
+                    PluginExecutionException(PluginExecutionException.INVALID_INPUT, "Error making POST request");
+        }
+        return response;
+    }
+
+    /**
+     * Makes a PUT request and returns the response
+     *
+     * @param url                       URL that the http call should be made to
+     * @param tokenPrefix               Prefix of the token of the request
+     * @param token                     Authorization token of the request
+     * @param contentType               Content type of the request
+     * @param payload                   Body of the request
+     * @throws PluginExecutionException Thrown when an exception is caught while the plugin executes
+     */
+    public HttpResponse makePutRequest(String url, String tokenPrefix, String token, String contentType,
+                                       StringEntity payload) throws PluginExecutionException {
+        HttpResponse response;
+        HttpClient httpClient = HttpClients.createDefault();
+        HttpPut putRequest = new HttpPut(url);
+        putRequest.setHeader("Authorization", tokenPrefix + " " + token);
+        putRequest.setHeader("Content-Type", contentType);
+        putRequest.setEntity(payload);
+        try {
+            response = httpClient.execute(putRequest);
+        } catch (IOException e) {
+            log.error("Error making PUT request", e);
+            throw new com.smartbear.swaggerhub.plugins.
+                    PluginExecutionException(PluginExecutionException.INVALID_INPUT, "Error making PUT request");
+        }
+        return response;
+    }
+
+    /**
+     * Makes a GET request and returns the response
+     *
+     * @param url                       URL that the http call should be made to
+     * @param tokenPrefix               Prefix of the token of the request
+     * @param token                     Authorization token of the request
+     * @param contentType               Content type of the request
+     * @return                          Returns the response of the request
+     * @throws PluginExecutionException Thrown when an exception is caught while the plugin executes
+     */
+    public HttpResponse makeGetRequest(String url, String tokenPrefix, String token, String contentType)
+            throws com.smartbear.swaggerhub.plugins.PluginExecutionException {
+        HttpResponse response;
+        HttpClient httpClient = HttpClients.createDefault();
+        HttpGet getRequest = new HttpGet(url);
+        getRequest.setHeader("Authorization", tokenPrefix + " " + token);
+        getRequest.setHeader("Content-Type", contentType);
```

Comment:
keeping these as constants would be better.

Reply:
Will do, thanks @dinusha92.